hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/FileContextMainOperationsBaseTest.java

```diff
@@ -975,6 +976,10 @@ public void testRenameFileAsExistingFile() throws Exception {
     
     // Succeeds with overwrite option
     rename(src, dst, true, false, true, Rename.OVERWRITE);
+
+    // Should not fail
```

Comment:
With our without this patch it never fail.
open call rely on RawLocalFs which never call the CheckSumFs open method

Reply:
Update your code with below code so it will call the method from CheckSumFs and ensure the crc is checked
```
FSDataInputStream in = fc.open(dst, 10);
    in.read();
    in.close();
```